### DISCUSSION TITLE
LG 32UN880 monitor often turns black when I increase or decrease the brightness

### DISCUSSION THREAD

**Discussion by alpennec** [MonitorControl/MonitorControl#1613] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Hello,

I've a really annoying issue: when I increase or decrease the brightness of my external monitor, the external monitor turns black. I need to switch it off and on again. I've observed that on macOS Sonoma and Sequoia.

My monitor is an LG 32UN880, connected to a MacBook Pro M2 with a USB-C cable.
I'm using MonitorControl 4.3.1-beta Build 7087.

Thanks,

### Steps to reproduce

I haven't found how to reproduce this issue manually. It's random.

### Expected behavior

I haven't found how to reproduce this issue manually. It's random.

### Environment Information (please complete the following information)

```markdown
- macOS version: 15.0 (24A335)
- Mac model: MacBook Pro Apple M2 Pro
- MonitorControl version: 4.3.1-beta Build 7087
- Monitor(s): LG 32UN880
- Apple Silicon/M1 (yes or no): M2 Pro
```

**Comment by waydabber** [comment]
This might be some monitor specific issue. Not sure about the reason. I'll convert this into a discussion, maybe others can chime in with the same display.

**Comment by szilardx** [comment]
Hi,
With LG 27UK650 I have the same issue. Upon controling brightness, monitor turns black. Have turn on off, or click on screen. Brightness does not change. Volume change results in same issues - and it does not work also.

The app looks nice anyway, it would be great to use.

**Reply by waydabber** [reply]
You might need to turn DDC off for the display if DDC control causes this issue. The app will fall back to software dimming then.
